argo/kfp-compiler/kfp_compiler/compiler.py

```diff
@@ -68,6 +68,10 @@ def load_fn(tfx_components: str, env: list):
     return fn
 
 
+def parse_namespaced_pipeline_name(namespaced_name: str) -> str:
+    return namespaced_name.replace("/", "-")
```

Comment:
would this affect the name of the Pipeline in the eventSource? i.e. if we had a Pipeline resource called `my/pipeline`, would the associated Sensor that triggers when the pipeline runs successfully need to match on `my-pipeline` or `my/pipeline`?
If the latter then worth updating the docs that `/` should be avoided in pipeline names. No pipelines have done that yet but could see it happening with a variant i.e. `pipeline/variant-training`

Reply:
I don't think its possible as K8s doesn't allow for `/` to be used, only `-` or `.`

Reply:
ok great